ArduCopter/mode_acro_heli.cpp

```diff
@@ -127,29 +127,29 @@ void ModeAcro_Heli::run()
 
 
 // virtual_flybar - acts like a flybar by leaking target atttitude back to current attitude
-void ModeAcro_Heli::virtual_flybar( float &roll_out_cds, float &pitch_out_cds, float &yaw_out_cds, float pitch_leak, float roll_leak)
+void ModeAcro_Heli::virtual_flybar( float &roll_out_rads, float &pitch_out_rads, float &yaw_out_rads, float pitch_leak, float roll_leak)
 {
-    Vector3f rate_ef_level_cds, rate_bf_level_cds;
+    Vector3f rate_ef_level_rads, rate_bf_level_rads;
 
     // get attitude targets
-    const Vector3f att_target_cd = attitude_control->get_att_target_euler_cd();
+    const Vector3f att_target_rad = attitude_control->get_att_target_euler_rad();
```

Comment:
it's an existing issue but this can be made a const reference.  so...

const Vector3f**&** att_target_rad = attitude_control->get_att_target_euler_rad();